app/src/main/java/com/example/meetup/Fragments/FeedFragment.java

```diff
@@ -0,0 +1,139 @@
+package com.example.meetup.Fragments;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.meetup.Adapters.PostsAdapter;
+import com.example.meetup.EndlessRecyclerViewScrollListener;
+import com.example.meetup.Models.Post;
+import com.example.meetup.R;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FeedFragment extends Fragment {
+
+    private static final String TAG = "FeedFragment";
+
+    protected RecyclerView posts;
+
+    public PostsAdapter adapter;
+    public List<Post> allPosts;
```

Comment:
It seems that both `adapter` and `allPosts` are only used in this class or the nested classes inside `FeedFragment`. We can make them `private`.